src/app/service/admin/CreateAFKRole.ts

```diff
@@ -0,0 +1,21 @@
+import ValidationError from '../../errors/ValidationError';
```

Comment:
We might not need the create AFK role command. 

The reasoning behind that is that it's much easier to create a role manually on discord. Actually that might be best because there are certain permissions that need to be set from @frogmonkee/AAJ 

Let's remove the create AFK command and instead we can provide you with the role ID that we can store in the source code